src/temp/AboutSection/AboutProject.tsx

```diff
@@ -0,0 +1,45 @@
+import styles from './AboutProject.module.scss';
+import { useState } from 'react';
+import { useTranslation } from 'next-i18next';
+import SeeMoreLessButton from './SeeMoreLessButton';
+
+interface Props {
+  description: string;
+  amountOfWords: number;
+}
+
+const AboutProject = ({ description, amountOfWords }: Props) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const splittedText = description.split(' ');
+
+  const itCanOverflow = splittedText.length > amountOfWords;
+  const beginText = itCanOverflow
+    ? splittedText.slice(0, amountOfWords - 1).join(' ')
```

Comment:
The logic for splitting the description into "beginText" and "endText" can be a bit convoluted. Consider simplifying it or adding comments to clarify its purpose.